dit/src/lib.rs

```diff
@@ -17,15 +20,55 @@ pub struct Args {
 pub enum Command {
     /// Starts the daemon.
     Daemon,
+    // Ping daemon
```

Comment:
```suggestion
    /// Pings the daemon.
```

Dann wird das für `dit --help` angezeigt.